2025-03-31 19:46:17,352 INFO [server.py:341] server listening on 0.0.0.0:10007
2025-03-31 19:46:21,565 INFO [server.py:529] connection open
2025-03-31 19:46:21,566 ERROR [server.py:378] connection handler failed
Traceback (most recent call last):
  File "/home/zdg/.conda/envs/sherpa-onnx/lib/python3.10/site-packages/websockets/asyncio/server.py", line 376, in conn_handler
    await self.handler(connection)
  File "/data/zdg/sherpa-onnx/test2.py", line 243, in <lambda>
    lambda ws: self.handle_connection(ws, translate_socket),
TypeError: StreamingServer.handle_connection() takes 2 positional arguments but 3 were given
2025-03-31 19:46:27,753 INFO [server.py:440] server closing
2025-03-31 19:46:27,754 INFO [server.py:473] server closed
